### PR TITLE
Add card UI with link support

### PR DESCRIPTION
**Pull request by zplata**
This task will add the UI for a Card display. Cards will display for successful service responses. And the design for now will be similar to that of the existing Sandbox card design. Along with this, we continue to add link support for absolute/smart apps, where for smart apps, we create a launch context if the Sandbox is running against a secured FHIR server. If the Sandbox is running against an open endpoint of a FHIR server, we simply append the `fhirServiceUrl` and `patientId` query parameter to the app, when it is launched. 

Suggestions will be implemented at a later PR.

Edit: Temporarily replacing the default CDS Services with a service that returns a SMART app.

@kpshek

### CODE REVIEW

File: src/reducers/cds-services-reducers.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import * as types from '../actions/action-types';
 
 const initialState = {
   configuredServices: {},
-  defaultUrl: 'https://fhir-org-cds-services.appspot.com/cds-services',
+  defaultUrl: 'http://cds2.interopion.com/cds-services',
```
Comment (kpshek) [inline src/reducers/cds-services-reducers.js]: Was this change intentional?

Comment (kpshek) [reply src/reducers/cds-services-reducers.js]: nm - I see in your description you say this is a temporary change

Comment (zplata) [reply src/reducers/cds-services-reducers.js]: Yeah, just edited the PR description. We'll change it back when it's done, but this is just to see a card that has a SMART app

**Review by kpshek — APPROVED** [review]

File: src/retrieve-data-helpers/launch-context-retrieval.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    };
+
+    if (link.appContext) {
+      launchParameters.appContext = link.appContext;
+    }
+
+    // May change when the launch context creation endpoint becomes a standard endpoint for all EHR providers
+    axios({
+      method: 'post',
+      url: `${fhirBaseUrl}/_services/smart/Launch`,
```
Comment (kolkheang) [inline src/retrieve-data-helpers/launch-context-retrieval.js:22]: just for my information, why using _ prefix here?

Comment (kolkheang) [reply src/retrieve-data-helpers/launch-context-retrieval.js:22]: also, why cap on the L?

File: src/components/Card/card.jsx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+  modifySmartLaunchUrls(card) {
+    return card.links.map((link) => {
+      let linkCopy = Object.assign({}, link);
+      if (link.type === 'smart' && this.props.fhirAccessToken) {
+        retrieveLaunchContext(
+          linkCopy, this.props.fhirAccessToken,
+          this.props.patientId, this.props.fhirServerUrl,
+        ).then((result) => {
+          linkCopy = result;
+          return link;
```
Comment (shriniketsarkar-zz) [inline src/components/Card/card.jsx]: Dont you want to return the updated `linkCopy` ?

**Review by kolkheang — APPROVED** [review]

**Review by shriniketsarkar-zz — APPROVED** [review]

File: src/retrieve-data-helpers/launch-context-retrieval.js (hunk 2 above)
Comment (zplata) [reply src/retrieve-data-helpers/launch-context-retrieval.js:22]: The default FHIR server for this Sandbox is HSPC. This was just the endpoint that they decided to use for registering a launch context for this scenario.
